### PR TITLE
SC: add unit tests for multi operator vt recovery

### PR DESCRIPTION
**Pull request by jeonghonah**
## Proposed changes

- Add following unit tests
    - value transfer recovery for multi-operator
    - a failure case for duplicated handle transfer that is already handled.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: node/sc/multi_bridge_test.go [hunk 1]
```diff
@@ -709,6 +709,35 @@ func TestMultiBridgeErrOverSign(t *testing.T) {
...
+	nonceOffset := uint64(17)
+	sentNonce := nonceOffset
+	transferAmount := uint64(100)
+	sentBlockNumber := uint64(100000)
+
+	tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
+	info.sim.Commit()
+	assert.NoError(t, bind.CheckWaitMined(info.sim, tx))
+
+	tx = SendHandleKLAYTransfer(info.b, acc, to, transferAmount, sentNonce, sentBlockNumber, t)
```
Comment (KimKyungup) [inline node/sc/multi_bridge_test.go]: I think If the different account case is added, it is useful. :)

File: node/sc/vt_recovery_test.go [hunk 2]
```diff
@@ -439,6 +439,110 @@ func TestScenarioAutomaticRecovery(t *testing.T) {
...
+		for i := 0; i < testTxCount; i++ {
+			ops[KLAY].request(info, info.localInfo)
+		}
+	})
+
+	// 2. Set multi-operator.
+	cAcc := info.nodeAuth
+	pAcc := info.chainAuth
+	opts := &bind.TransactOpts{From: cAcc.From, Signer: cAcc.Signer, GasLimit: testGasLimit}
+	_, err := info.localInfo.bridge.RegisterOperator(opts, pAcc.From)
```
Comment (KimKyungup) [inline node/sc/vt_recovery_test.go]: for local, cAcc?
for remote, pAcc?

File: node/sc/vt_recovery_test.go [hunk 3]
```diff
@@ -439,6 +439,110 @@ func TestScenarioAutomaticRecovery(t *testing.T) {
...
+	}
+
+	// 6. Check pending events.
+	t.Log("check pending tx", "len", len(vtr.serviceChainEvents))
+	var count = 0
+	for _, ev := range vtr.serviceChainEvents {
+		assert.Equal(t, info.nodeAuth.From, ev.From)
+		assert.Equal(t, info.aliceAuth.From, ev.To)
+		assert.Equal(t, big.NewInt(testAmount), ev.ValueOrTokenId)
+		assert.Condition(t, func() bool {
```
Comment (KimKyungup) [inline node/sc/vt_recovery_test.go]: if possible??
```suggestion
		assert.Condition(t, uint64(testBlockOffset) <= ev.Raw.BlockNumber)
```

File: node/sc/multi_bridge_test.go (hunk 1 above)
Comment (jeonghonah) [reply node/sc/multi_bridge_test.go]: Added.

File: node/sc/vt_recovery_test.go (hunk 2 above)
Comment (jeonghonah) [reply node/sc/vt_recovery_test.go]: It's an intentional account mix to register different operators.

File: node/sc/vt_recovery_test.go (hunk 3 above)
Comment (jeonghonah) [reply node/sc/vt_recovery_test.go]: assert.Condition() accepts a function only.

**Pull request merged**